scripts/train.py

```diff
@@ -0,0 +1,181 @@
+from torch.optim import Adam
+from torch import nn, optim
+import torch
+import math
+import time
+import os
+import sys
+
+
+'''
+Decouple the reference path.
+'''
+cwd = os.getcwd()  # current working directory
+cfp = os.path.dirname(os.path.abspath(__file__))  # current file path
+os.chdir(cfp)
+sys.path.append(os.path.abspath(".."))
+os.chdir(cwd)
+
+from nn_modules.transformer.top_former import TopFormer
+from utils.config import top_config
+from data.data_loader import train_loader, val_loader
+
+model = TopFormer(num_layers=top_config["model_params"]["num_layers"], d_model=top_config["model_params"]
+                  ["d_model"], nhead=top_config["model_params"]["nhead"], num_classes=top_config["model_params"]["num_classes"])
+
+# TODO(diwei): remove
+
+
+def count_parameters(model):
+    return sum(p.numel() for p in model.parameters() if p.requires_grad)
+
+
+# TODO(diwei): remove
+def initialize_weights(m):
```

Comment:
What does `m` mean here? Model?

Consider make it consistent, e.g., `model`.

Reply:
Done